doc/role_whitelist_management.md

```diff
@@ -246,6 +268,24 @@ without any record of that in the audit log.
 cassandra@cqlsh> ALTER ROLE ibbe WITH OPTIONS = { 'GRANT AUDIT WHITELIST FOR ALTER' : 'roles/jim' };
 ```
 
+### Grant Resources
+
+This resource type is used to represent an access derived whitelist on the wrapped resource.
```

Comment:
Now when I'm reading this, I'm having second thoughts on the name we selected for this concept "access derived whitelist". In this sentence it can be interpreted as if the whitelist is derived by the action of accessing a resource - or at least it is not clear what we're trying to say. Would perhaps "permission derived" work better?

Reply:
yes, changed to "permission derived"